scripts/shellby/src/index.ts

```diff
@@ -0,0 +1,23 @@
+import * as childProcess from 'child_process'
+
+const parseCommand = (cmd: string): { command: string; args: string[] } => {
+  const [command, ...args] = cmd.split(' ')
+  if (command) {
+    return {
+      command,
+      args,
+    }
+  }
+  return {
```

Comment:
If `command` is falsy then it means `cmd` was an empty string, we should probably throw an error instead?

Reply:
dang you're right, In my head in meant the command had no spaces